kin/blockchain/builder.py

```diff
@@ -40,9 +38,16 @@ def clear(self):
         self.tx = None
         self.te = None
 
-    def get_sequence(self):
-        """Alternative implementation to expose exceptions"""
-        return self.horizon.account(self.address).get('sequence')
+    def update_sequence(self):
+        """
+        Update the builder with the *current* sequence of the account
+        # TODO: kin-base builder increments this value by 1 when building a tx.
+        #       Remove this functionality from py-stellar-base and change this method set the current sequence+1
+        """
+
+        # TODO: kin-base checks for 'not sequence' to find if there is no sequence, therefore
+        # Sequence of 0 fails, write it as a str for now and fix in kin-base later
```

Comment:
aren't you using a default of `sequence = 1` though? Why do you need this hack then?

Reply:
If you use the root account, its first sequence will be 0. kin-base will then do:
```if not self.sequence:
            raise SequenceError('No sequence is present, maybe not funded?')
```
So a sequence if 0 will raise an error. That's why I pass it a '0'